### DISCUSSION TITLE
Alpine huts default zoom level

### DISCUSSION THREAD

**Discussion by Albynton** [organicmaps/organicmaps#6055] [Ideas]
When hiking in the mountains, it is quite common to go from one mountain hut (`alpine_hut` in OSM) to another. Both when planning a trip and when hiking, it is important to be able to see the huts without having to zoom in too much. This allows you to see the surrounding terrain.

Currently, the hut POIs are only visible at a high zoom level (z16 if I understand the style file correctly).

It would be useful to have the hut icon visible when the map is not so zoomed in, as it would then be easier to locate them. I suggest moving it to a zoom of z15 or z14 (as I understand it this should be equivalent to the drinking water icon). Since the high mountain areas are quite barren, this should not clutter the map.

For reference, both OSM Carto and OpenTopoMaps show the hut icon from level 13.

What do you think?

**Comment by biodranik** [comment]
1. We can try, it would be great to see screenshots in different areas and decide if they are too polluted or not for the _generic_ map style.
2. As we're planning an #2356 Outdoors style, there alpine huts definitely can be seen earlier.

**Comment by Albynton** [comment]
Here are screenshots of three different huts in the Alps. I tried to zoom back as much as possible while keeping the icon displayed.

The respective coordinates are:
1. Austria: 47.1212487, 9.644521
2. Italy: 45.8499383, 7.2691821
3. Switzerland: 46.0922603, 7.6333041

![Austria-small](https://github.com/organicmaps/organicmaps/assets/26968739/98b92987-0dc5-45e2-b83c-94db4a0770f6)
![Italy-small](https://github.com/organicmaps/organicmaps/assets/26968739/f5c4484a-0921-4b6a-9265-1350905d1089)
![Switzerland-small](https://github.com/organicmaps/organicmaps/assets/26968739/2ae98abd-2dec-4428-89ea-c8bc568d51b4)

**Comment by map-per** [comment]
I made a PR for this (#6112)

**Reply by Albynton** [reply]
Thanks a lot!

**Reply by map-per** [reply]
It will be in the next release 🎉

**Reply by Albynton** [reply]
Awesome! Thank you for your involvement, it'll greatly help planning mountain trips.
